src/player/Player.vue

```diff
@@ -68,6 +68,13 @@
                         :value="volume" @input="setVolume"
                 />
               </b-popover>
+              <b-button title="ReplayGain"
+                        variant="link" class="m-0"
+                        @click="toggleReplayGain">
+                <IconReplayGain v-if="replayGainMode === ReplayGainMode.None" />
+                <IconReplayGainTrack v-else-if="replayGainMode === ReplayGainMode.Track" />
+                <IconReplayGainAlbum v-else-if="replayGainMode === ReplayGainMode.Album" />
+              </b-button>
```

Comment:
Should probably be disabled if track doesn't have ReplayGain?

Reply:
good idea, I'll add that